### PR TITLE
revert: "feat(portal): Add 'temp account' feature for launch HN"

### PR DESCRIPTION
**Pull request by jamilbk**
Reverts firezone/firezone#6153

### CODE REVIEW

File: elixir/apps/web/lib/web/components/core_components.ex [hunk 1]
```diff
@@ -285,10 +285,10 @@ defmodule Web.CoreComponents do
       id={@id}
       class={[
         "p-4 text-sm flash-#{@kind}",
-        @kind == :success && "text-green-800 bg-green-100",
-        @kind == :info && "text-blue-800 bg-blue-100",
-        @kind == :warning && "text-yellow-800 bg-yellow-100",
-        @kind == :error && "text-red-800 bg-red-100",
```
Comment (bmanifold) [inline elixir/apps/web/lib/web/components/core_components.ex]: This was actually one change I was hoping we might be able to keep

File: elixir/apps/web/lib/web/router.ex [hunk 2]
```diff
@@ -66,12 +66,6 @@ defmodule Web.Router do
     live "/", SignUp
   end
 
-  scope "/try", Web do
-    pipe_through :public
-
-    live "/", TempAccounts.Index
-  end
```
Comment (bmanifold) [inline elixir/apps/web/lib/web/router.ex]: Was wondering if we should keep this path and just have it redirect to `/sign_up`?  Just thinking if anyone comes across the Launch HN then at least that link will still work technically.  We could even have a small delay saying "Temporary accounts have been removed, but you can get started with a free account here"

**Review by bmanifold — COMMENTED** [review]
Overall, looks good, just a couple of minor recommendations/questions.

File: elixir/apps/web/lib/web/components/core_components.ex [hunk 3]
```diff
@@ -285,10 +285,10 @@ defmodule Web.CoreComponents do
...
+        @kind == :success && "text-green-800 bg-green-50",
+        @kind == :info && "text-blue-800 bg-blue-50",
+        @kind == :warning && "text-yellow-800 bg-yellow-50",
+        @kind == :error && "text-red-800 bg-red-50",
```
Comment (jamilbk) [inline elixir/apps/web/lib/web/components/core_components.ex]:
```suggestion
        @kind == :success && "text-green-800 bg-green-100",
        @kind == :info && "text-blue-800 bg-blue-100",
        @kind == :warning && "text-yellow-800 bg-yellow-100",
        @kind == :error && "text-red-800 bg-red-100",
```

File: elixir/apps/web/lib/web/router.ex (hunk 2 above)
Comment (jamilbk) [reply elixir/apps/web/lib/web/router.ex]: Yeah good call, I'll just redirect to `/sign_up`.

File: elixir/apps/web/lib/web/router.ex [hunk 4]
```diff
@@ -66,12 +66,6 @@ defmodule Web.Router do
     live "/", SignUp
   end
```
Comment (jamilbk) [inline elixir/apps/web/lib/web/router.ex:68]:
```suggestion
    # Maintained from the LaunchHN - show SignUp form
    scope "/try", Web do
      pipe_through :public

      live "/", SignUp
    end
```

**Review by bmanifold — COMMENTED** [review]
Just tried this locally.  If we keep the temp accounts in the DB and someone tries to access their temp account it will result in a crash, because it's trying to use a provider that doesn't exist.  Not sure if that's a concern, but wanted to give a heads up.

**Comment by jamilbk** [comment]
> Just tried this locally. If we keep the temp accounts in the DB and someone tries to access their temp account it will result in a crash, because it's trying to use a provider that doesn't exist. Not sure if that's a concern, but wanted to give a heads up.

Yeah we should probably purge those when this ships. Is there an easy way to query them?

**Comment by bmanifold** [comment]
> Is there an easy way to query them?

Yep.  Just shared a doc with you that has all the temp account IDs/names/slugs.

**Review by bmanifold — APPROVED** [review]
LGTM.  As discussed in this PR, we just need to make sure we scrub the DB prior to deploying these changes.

**Review by AndrewDryga — APPROVED** [review]

**Pull request merged**